contrib/!lang/ruby/packages.el

```diff
@@ -37,7 +37,16 @@
   "Initialize Ruby Mode"
   (use-package enh-ruby-mode
     :mode (("\\(Rake\\|Thor\\|Guard\\|Gem\\|Cap\\|Vagrant\\|Berks\\|Pod\\|Puppet\\)file\\'" . enh-ruby-mode)
-           ("\\.\\(rb\\|rabl\\|ru\\|builder\\|rake\\|thor\\|gemspec\\|jbuilder\\)\\'" . enh-ruby-mode))))
+           ("\\.\\(rb\\|rabl\\|ru\\|builder\\|rake\\|thor\\|gemspec\\|jbuilder\\)\\'" . enh-ruby-mode))
+    :config
+    (progn
+      (setq enh-ruby-deep-indent-paren nil
+            enh-ruby-hanging-paren-deep-indent-level 2)
```

Comment:
This variable handles the case of a method with many parameters, indenting it like this:
```ruby
def lol(hi, wow, troll,
  wat, thounh)
```
Instead of:
```ruby
def lol(hi, wow, troll,
wat, thounh)
```